Week2/6_arry.js

```diff
@@ -0,0 +1,8 @@
+let zoo=[];
+console.log(zoo);
+console.log("Animals that I saw in the Zoo");
```

Comment:
Is this the value of `zoo`? I think that in the assignment you have to write down the value

Reply:
I got confused about this points.
>  According to the assignment i have to write what the value of the array is.
> But on this step (6.2) the array is still empty and if i run **console.log(zoo)** the result will be [ ]
So is this what i have to write as value of the array ?

Reply:
Yeah, exactly! That is the value